source/_integrations/qnap.markdown

```diff
@@ -118,9 +118,9 @@ monitored_conditions:
       description: Displays the used space of the volume as a percentage (creates a new entry for each volume).
 {% endconfiguration %}
 
-### Self-signed certificates
+### SSL & Self-signed certificates
```

Comment:
```suggestion
### SSL and Self-signed certificates
```

Reply:
Thank you @frenck, for you super-fast & helpful feedback!